python/apps/taiga/src/taiga/notifications/commands.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2023-present Kaleidos INC
+
+from datetime import timedelta
+
+import typer
+from taiga.base.utils import pprint
+from taiga.base.utils.concurrency import run_async_as_sync
+from taiga.base.utils.datetime import aware_utcnow
+from taiga.conf import settings
+from taiga.notifications import services as notifications_services
+
+cli = typer.Typer(
+    name="Taiga Notifications commands",
+    help="Manage the notifications system of Taiga.",
+    add_completion=True,
+)
+
+
+@cli.command(help="Clean read notifications. Remove entries from DB.")
+def clean_read_notifications(
+    minutes_to_store_read_notifications: int = typer.Option(
+        settings.NOTIFICATIONS.MINUTES_TO_STORE_READ_NOTIFICATIONS,
+        "--minutes",
+        "-m",
+        help="Number of minutes to store read notifications",
```

Comment:
I think this message it's not really clear. It seems to be changing the settings instead of deleting the notifications read before that time. 
![image](https://github.com/taigaio/taiga/assets/4539017/f4e173cb-c249-4d2f-9309-e0e95f09e3a8)

I can imagine a message similar to this one: "Delete every notification read before the specified minutes"

Reply:
The default time taken from the settings works fine and it's very cool ;)